manim/__main__.py

```diff
@@ -123,7 +124,9 @@ def get_scene_classes_from_module(module):
 def get_module(file_name):
     if file_name == "-":
         module = types.ModuleType("input_scenes")
+        logger.info("Enter the code for your animation and end with an EOF (usually CTRL+D):")
         code = sys.stdin.read()
+        logger.info("Rendering animation from typed code...")
         try:
```

Comment:
Didn't there used to be a line here that added `from manim import *` to the top of the file? Would be better to keep it that way imo since I see this option being used mostly for quick testing and prototyping manim scenes so adding a `from manim import *` everytime seems redundant.

Reply:
Huh. Don't recall seeing that. In any case I'll add it in in a few hours, once I'm awake.

Reply:
Done.